docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -1502,12 +1502,23 @@ The only particular aspect of writing Quarkus extensions in Eclipse is that APT
 
 The class augmentation step of Quarkus generates classes for various purposes. Sometimes you need to view these
 classes/bytecode to debug or understand an issue. Classes that are related to application augmentation are currently held in
-memory in a runtime class loader. To have these classes written out to disk for inspection, specify the
+memory in a runtime class loader.
```

Comment:
I think we should either mention which classes  are in `target/wiring-devmode`, or not mention this at all

Reply:
if you know what's in it I can add it :)

Reply:
I would need to check and I am currently very short on time 😬

Reply:
The clue is in the source code:

```
            //wiring devmode is used for CDI beans that are not part of the user application (i.e. beans in 3rd party jars)
            //we need this because these beans cannot be loaded by the runtime class loader, they must be loaded by the platform
            //class loader
            File wiringClassesDirectory = new File(buildDir, "wiring-devmode");
```

So I think this should be skipped entirely since it's of no interest to extension authors.

Reply:
OK, I remove the sentence about the `target/wiring-devmode` directory